backend/git_real/tests/test_activity_log_creators.py

```diff
@@ -102,4 +106,43 @@ def test_pushing(self, has_permission):
         self.assertEqual(entry.effected_user, push.repository.user)
         self.assertEqual(entry.object_1, push)
         self.assertEqual(entry.object_2, push.repository)
-        self.assertEqual(entry.event_type.name, creators.GIT_PUSH)
\ No newline at end of file
+        self.assertEqual(entry.event_type.name, creators.GIT_PUSH)
+
+
+class log_pr_opened_Tests(APITestCase):
+    @mock.patch.object(IsWebhookSignatureOk, "has_permission")
+    def test_pr_opened(self, has_permission):
+        has_permission.return_value = True
+
+        self.assertEqual(PullRequest.objects.count(), 0)
+
+        body, headers = get_body_and_headers("pull_request_opened")
+        pull_request_data=body["pull_request"]
+        url = reverse(views.github_webhook)
+
+        github_name = pull_request_data["user"]["login"]
+        SocialProfileFactory(github_name=github_name)
+        user = User.objects.filter(social_profile__github_name__iexact=github_name).first()
```

Comment:
this query seems very complicated.

Why not:

wouldn't this work?
```
social_profile = SocualPRofileFactory(...)
user = social_profile.user
```

or even `User.objects.get(social_profile = social_profile)`